src/AppBundle/Controller/Api/CommitteesController.php

```diff
@@ -25,8 +27,12 @@ public function getApprovedCommitteesAction()
      * @Route("/events", name="api_committees_events")
      * @Method("GET")
      */
-    public function getUpcomingCommitteesEventsAction()
+    public function getUpcomingCommitteesEventsAction(Request $request)
     {
-        return new JsonResponse($this->get('app.api.event_provider')->getUpcomingEvents());
+        try {
+            return new JsonResponse($this->get('app.api.event_provider')->getUpcomingEvents($request->query->get('type')));
+        } catch (\Exception $e) {
```

Comment:
Please catch `EventException` only, we don't want bad request code for anything else.